velox/core/Config.cpp

```diff
@@ -41,6 +43,19 @@ folly::Optional<std::string> MemConfigMutable::get(
   return val;
 }
 
+int16_t MemConfig::sessionTimezoneId() const {
```

Comment:
Let's keep this change limited to just validating the config values. 

Let's then follow-up to add this getter, but make sure it doesn't parse and check the timezone on each call. We'd want to initialize a member variable when timezone is set once and then just return the integer.

Reply:
Makes sense. Updated this PR to keep limited change.